src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUsersById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentById(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
+  );
+}
+
+const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUsersById(post.userId),
+  comments: getCommentById(post.id),
+}));
```

Comment:
You can make from this clear function, and move it to a separate file ( for future )